### PR TITLE
Fix titles in web navigation

### PR DESCRIPTION
**Pull request by Lennonka**
Applying sentence case to upstream titles, especially in navigation.
Supported releases and nightly only.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into: N/A

### CODE REVIEW

File: web/releases/nightly.json [hunk 1]
```diff
@@ -115,83 +115,83 @@
       "filename": "index-katello.html",
       "guides": [
         {
-          "title": "Release Notes",
+          "title": "Release notes",
           "path": "Release_Notes"
         },
         {
-          "title": "Planning Guide",
+          "title": "Planning guide",
```
Comment (maximiliankolb) [inline web/releases/nightly.json:122]:
```suggestion
          "title": "Planning for Foreman with Katello",
```

See https://github.com/theforeman/foreman-documentation/pull/3195/files#diff-7c56a5e93aac7a02f5ff3b7c231a4573ce1dbd6a9d5f6e3c6979cf393d566431R54

**Review by maximiliankolb — APPROVED** [review]
We also use https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L124 aka "Foreman Server" in favor of "Foreman server" but I am OK either way.

One small suggestion, rest of the diff LGTM. Hard to say if something was missed. I suggest to merge and then check the actual docs on docs.theforeman.org.

File: web/releases/nightly.json (hunk 1 above)
Comment (Lennonka) [reply web/releases/nightly.json:122]: Yes, what should I notice in particular?

Comment (Lennonka) [reply web/releases/nightly.json:122]: Oh, I see now :)

**Review by maximiliankolb — APPROVED** [review]
LGTM.

**Pull request merged**